### PR TITLE
Download data if missing.

### PR DESCRIPTION
**Pull request by andreibancioiu**
Download missing data, if instructed to do so.
 - `export DOWNLOAD_REGULAR_ARCHIVE=yes or true`: will download a regular daily archive
 - `export DOWNLOAD_NON_PRUNED_EPOCHS=yes or true`: will download epochs with non-pruned state, as specified by `EPOCH_FIRST`, `EPOCH_LAST`.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -2,30 +2,133 @@
...
+    fi
+
+    if [[ -z "${EPOCH_LAST}" ]]; then
+        echo "Error: EPOCH_LAST isn't set."
+        return 1
+    fi
+
+    mkdir -p db/${CHAIN_ID}
+
+    echo "Downloading Static.rar ..."
```
Comment (iulianpascalau) [inline entrypoint.sh]: rar? Should have been tar :D

File: entrypoint.sh [hunk 2]
```diff
@@ -2,30 +2,133 @@
...
+    wget ${NON_PRUNED_URL_BASE}/Static.tar || return 1
+
+    for (( epoch = ${EPOCH_FIRST}; epoch <= ${EPOCH_LAST}; epoch++ ))
+    do
+        echo "Downloading Epoch_${epoch}.tar ..."
+        wget ${NON_PRUNED_URL_BASE}/Epoch_${epoch}.tar || return 1
+    done
+
+    echo "Extracting Static.rar"
```
Comment (iulianpascalau) [inline entrypoint.sh]: tar

File: entrypoint.sh [hunk 3]
```diff
@@ -2,30 +2,133 @@
...
+    tar -xf Static.tar --directory db/${CHAIN_ID} || return 1
+
+    echo "Removing Static.rar"
```
Comment (iulianpascalau) [inline entrypoint.sh]: tar

File: entrypoint.sh (hunk 1 above)
Comment (andreibancioiu) [reply entrypoint.sh]: Fixed (and below).

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**